README.md

```diff
@@ -33,20 +44,42 @@ on:
 jobs:
   lints:
     name: Build
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-latest
     permissions:
       id-token: "write"
       contents: "read"
     steps:
       - uses: actions/checkout@v3
-      - name: Install Nix
-        uses: DeterminateSystems/nix-installer-action@main
+      - uses: DeterminateSystems/nix-installer-action@main
         with:
           flakehub: true
-      - name: Run `nix build`
-        run: nix build .
+      - run: nix build .
 ```
 
+See [`.github/workflows/ci.yml`](.github/workflows/ci.yml) for a full example.
+
+### Advanced Usage
+
+* If KVM is available, the installer exports the `DETERMINATE_NIX_KVM` environment variable set to 1.
```

Comment:
```suggestion
* If KVM is available, the installer sets up KVM so that Nix can detect it, and exports the `DETERMINATE_NIX_KVM` environment variable set to 1.
```